build.sh

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
+rm -f /etc/apt/apt.conf.d/docker-clean
+echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
+
 apt-get update
 apt-get install -y git
-pip install --only-binary=scrypt --no-cache-dir -e .
+
+pip install --only-binary=scrypt --disable-pip-version-check install .
```

Comment:
Why do we switch away from the editable install?

Reply:
Good question, I am not sure anymore... I will check what happens if I turn it back to an editable install

Reply:
I can't find out anymore, why I did it, so I "reverted" that.